test/user-search.ts

```diff
@@ -0,0 +1,153 @@
+    })
+
+    app.use(requestLogger(logger))
+    app.use(rpcLogger(logger))
+    router.post('/', rpc.middleware)
+    app.use(router.routes())
+    rpc.register('get_account', userSearch.getAccount)
+    rpc.register('autocomplete_account', userSearch.autocompleteAccount)
+    return app
+}
+
+describe('user search functions', function() {
+    this.slow(1000)
+    this.timeout(10 * 1000)
+
+    beforeEach(function(this) {
+        const fakeSteemdClient = {
+            call: async function fakeCall(
+                api: string,
+                method: string,
+                params: any[]
+            ) {
+                if (method === 'get_accounts') {
+                    return [getAccountsResponse]
+                }
+                if (method === 'get_follow_count') {
+                    return getFollowCountResponse
+                }
+                if (method === 'get_followers' && params[2] === 1) {
+                    return getFollowersResponse
+                }
+                if (method === 'get_followers' && params[2] === 2) {
+                    return getIgnoredResponse
+                }
+                if (method === 'get_following') {
+                    return getFollowingResponse
+                }
+                if (method === 'get_account_history') {
+                    return getAccountHistoryResponse
+                }
+                if (method === 'lookup_accounts') {
+                    return lookupAccountsResponse
+                }
+            }
+        }
+        const cache = new nodecache({ stdTTL: 600, checkperiod: 60 })
+        this.fakeCacheClient = new CachingClient(
+            cache,
+            { stdTTL: 600, checkperiod: 60 },
+            fakeSteemdClient
+        )
+    })
+
+    it('should return an account', async function() {
+        const app = createApp(this.fakeCacheClient)
+        const response = await request(app.callback()).post('/')
+            .set('Content-Type', 'application/json')
+            .send({id: 1, jsonrpc: '2.0', method: 'conveyor.get_account', params:{account: 'steemit'}})
+        const jsonRPCResponse = JSON.parse(response.text)
+        const result = jsonRPCResponse.result
+        assert.deepEqual(result, expectedUserAccountJSON)
+    })
+    it('should return account suggestions', async function() {
+        const app = createApp(this.fakeCacheClient)
+        const acctNames = new Set(['steemit', 'steemit2', 'aaronburt'])
+        app.context.cacheClient = this.fakeCacheClient
+        app.context.userAccountTrie = new AccountNameTrie(acctNames, this.fakeCachingClient, 100000)
+        const response = await request(app.callback()).post('/')
+            .set('Content-Type', 'application/json')
+            .send({id: 1,
+                      jsonrpc: '2.0',
+                      method: 'conveyor.autocomplete_account',
+                      params: {accountSubstring: 'steemi', account: 'steemit'}})
```

Comment:
Based on `user/search.ts`, it seems like `account` here needs to be `contextAccount`.